scriptworker/cot/verify.py

```diff
@@ -2225,3 +2226,80 @@ def verify_cot_cmdln(args=None, event_loop=None):
             rm(tmp)
         else:
             log.info("Artifacts are in {}".format(tmp))
+
+
+# create_test_workdir {{{1
```

Comment:
What does the `{{{1` represent? Does it format the pydoc? :thinking:

Reply:
It's a vim folding marker. Not everyone uses vim, but it's relatively harmless comment for other editors. This lets me group functions to fold, making the 2305 line file easier to navigate.